Automattic/jetpack
# Action Bar: initial scaffolding for package and module

#### Changes proposed in this Pull Request:

Scaffolds a new package for the Action Bar, allowing site visitors to follow, like, and comment on a site.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/1699996/184253153-361f6fa1-9d1b-4e33-bfed-ec022e888ed6.png">


#### Other information:


#### Jetpack product discussion

pe7F0s-1U-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

- Switch to this branch and run `jetpack install --all`
- Build the action-bar package `jetpack build packages/action-bar`
- Go to `/wp-admin/admin.php?page=jetpack_modules` and enable the Action bar module
- Go the a single post page and you should see the action bar html in the lower left of the window

Makes a start at resolving https://github.com/Automattic/jetpack/issues/6848
